src/portal/src/lib/components/create-edit-endpoint/create-edit-endpoint.component.spec.ts

```diff
@@ -198,7 +198,16 @@ describe("CreateEditEndpointComponent (inline template)", () => {
           {
             "key": "sa-east-1",
             "value": "https://api.ecr.sa-east-1.amazonaws.com"
-          }
+	  },
+	  {
+	    "key": "cn-north-1",
+	    "value": "https://api.ecr.cn-north-1.amazonaws.com.cn"
+	  },
+	  {
+	    "key": "cn-northwest-1",
+	    "value": "https://api.ecr.cn-northwest-1.amazonaws.com.cn"
+	  }
```

Comment:
The changes for UI codes isn't necessary, UI should render the endpoints automatically if you change the backend codes.